sky/packages/sky/lib/src/animation/animated_value.dart

```diff
@@ -15,7 +15,10 @@ enum Direction {
   reverse
 }
 
-/// A variable that changes as an animation progresses
+/// An interface describing a variable that changes as an animation progresses.
+/// 
+/// AnimatedVariables, by convention, must be cheap to create. This allows them to be used in 
+/// build functions in Widgets.
```

Comment:
Is that true? I've used them in build functions for the case where build is not going to be called every frame (the are passed to a Transition that animates). Your change preserves that property, right?